elasticdl/docker/Dockerfile.ci

```diff
@@ -13,22 +14,22 @@ RUN python -m pip install --quiet -r /model_zoo/requirements.txt \
         --extra-index-url=${EXTRA_PYPI_INDEX}
 
 # Install elasticdl_preprocessing package
-COPY build/elasticdl_preprocessing-develop-py3-none-any.whl /
-RUN python -m pip install --quiet /elasticdl_preprocessing-develop-py3-none-any.whl \
+COPY build/elasticdl_preprocessing-{PKG_VERSION}-py3-none-any.whl /
```

Comment:
Maybe we can copy all the wheel files into the image, and install them in one line, multiple `RUN`, minimize the layers of Docker image can reduce the size.
``` docker
COPY build/*.whl /
RUN pip install /*.whl && rm /*.whl
```

c.f. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers